### PR TITLE
docs(python): add custom error page to Python docs

### PR DESCRIPTION
**Pull request by Anurag-modgil**
### Remarks

PR introduces (a) breaking change(s): no

PR introduces (a) deprecation(s): no

### CODE REVIEW

File: streampipes-client-python/mkdocs.yml [hunk 1]
```diff
@@ -106,3 +106,8 @@ nav:
       - Exploring Live Data from a StreamPipes Data Stream: tutorials/3-getting-live-data-from-the-streampipes-data-stream.ipynb
       - Applying Online Machine Learning on a StreamPipes Data Stream: tutorials/4-using-online-machine-learning-on-a-streampipes-data-stream.ipynb
   - 📚 Reference: reference/*
+  theme:
+  name: material
```
Comment (bossenti) [inline streampipes-client-python/mkdocs.yml:110]: This is a duplicate of line 29

File: streampipes-client-python/docs/overrides/main.html [hunk 2]
```diff
@@ -22,6 +22,12 @@
 <!-- More minimal footer -->
 {% block footer %}
 <footer class="md-footer">
+    <div>{% extends "base.html" %}
+
+        {% block htmltitle %}
+          <title>Lorem ipsum dolor sit amet</title>
```
Comment (bossenti) [inline streampipes-client-python/docs/overrides/main.html:28]: Please remove this

File: streampipes-client-python/overrides/text.docs [hunk 3]
```diff
@@ -0,0 +1,41 @@
+├─ .icons/                             # Bundled icon sets
```
Comment (bossenti) [inline streampipes-client-python/overrides/text.docs:1]: I don't think this file is required

**Review by bossenti — CHANGES_REQUESTED** [review]
@Anurag-modgil thanks for your pr 🙏 I have marked some aspects for improvement

File: streampipes-client-python/mkdocs.yml (hunk 1 above)
Comment (Anurag-modgil) [reply streampipes-client-python/mkdocs.yml:110]: should i remove this or replace it

Comment (bossenti) [reply streampipes-client-python/mkdocs.yml:110]: You can remove it, since it doesn't add any value

Comment (Anurag-modgil) [reply streampipes-client-python/mkdocs.yml:110]: already done thanks for feedback

Comment (bossenti) [reply streampipes-client-python/mkdocs.yml:110]: Have you updated the PR? It's still contained in the diff
Same applies for the other aspects I've put a comment on

File: streampipes-client-python/overrides/text.docs [hunk 4]
```diff
@@ -0,0 +1,40 @@
+├─ assets/
```
Comment (bossenti) [inline streampipes-client-python/overrides/text.docs:1]: From my point of view the whole `text.docs` file is not required
Or what is the reason why you include it?

File: streampipes-client-python/docs/overrides/404.html [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+    <img src="https://media.giphy.com/media/26tk0W5u5x5KZHJZu/giphy.gif" alt="Page not found">
+    <p>Oops! Looks like you took a wrong turn.</p>
+    <p>But don't worry, we're here to help.</p>
+    <p>Please try one of these links:</p>
+    <ul>
+      <li><a href="/">Homepage</a></li>
+      <li><a href="/blog">Blog</a></li>
+      <li><a href="/contact">Contact Us</a></li>
+    </ul>
+  </body>
```
[the remark this answers is not in the document]
Comment (Anurag-modgil) [reply streampipes-client-python/docs/overrides/404.html]: thanks for the help and sorry for delayed response

[the remark this answers is not in the document]
Comment (bossenti) [reply streampipes-client-python/docs/overrides/404.html]: No worries
The code snippet I've posted should be the only content of `404.html`
Have you ever build the docs with your version of the error page? 
With defining your own style, this does not suit to the appearance of the remaining docs.
As long the error page does not fit our style, we won't merge the PR
Sorry about that

**Pull request closed without merging**